pkg/resources/mutation/mutation.go

```diff
@@ -1,20 +1,26 @@
 package mutation
 
 import (
+	"fmt"
+
+	"github.com/rancher/webhook/pkg/admission"
 	"github.com/rancher/webhook/pkg/auth"
 	"github.com/rancher/webhook/pkg/patch"
-	"github.com/rancher/wrangler/pkg/webhook"
+	v1 "k8s.io/api/admission/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-func SetCreatorIDAnnotation(request *webhook.Request, response *webhook.Response, obj runtime.Object, newObj metav1.Object) error {
+// SetCreatorIDAnnotation sets the creatorID Annotation on the newObj based  on the user specified in the request.
+func SetCreatorIDAnnotation(request *admission.Request, response *v1.AdmissionResponse, obj runtime.RawExtension, newObj metav1.Object) error {
 	annotations := newObj.GetAnnotations()
 	if annotations == nil {
 		annotations = map[string]string{}
 	}
 	annotations[auth.CreatorIDAnn] = request.UserInfo.Username
 	newObj.SetAnnotations(annotations)
-
-	return patch.CreatePatch(obj, newObj, response)
+	if err := patch.CreatePatch(obj.Raw, newObj, response); err != nil {
```

Comment:
It really shouldn't be. I updated this code in a separate norman -> steve migration PR instead of this one because it focuses on the mutators that use this function. That said, the new PR won't be up this release, so I can move the changes here.